applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -77,6 +62,59 @@ Make sure that the DFU lock utility is enabled if your nRF Desktop application c
 You cannot use this module with the :ref:`caf_ble_smp`.
 In other words, you cannot simultaneously enable the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option and the :kconfig:option:`CONFIG_CAF_BLE_SMP` Kconfig option.
 
+You have to choose backend for the DFU process by enabling one of the following options:
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT` - This option depends on the MCUboot bootloader support for the DFU process.
+    For details, see :ref:`mcuboot` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_SUIT` - This option depends on the SUIT tool support for the DFU process.
```

Comment:
As in other comment: "I do not recall seeing SUIT being called "tool" in other docs. IMO it will be better to just leave it named "SUIT"."